examples/gpt-2/README.md

```diff
@@ -122,7 +136,56 @@ in this way".
 
 ## Quick Start (II) - Fine-tune the Pre-trained Model 
 
-Coming soon!
+This section shows how we can fine-tune the pre-trained GPT2 model and use the resulting model for generation.
+
+### Prepare data
+
+We first preprocess data with the GPT-2 BPE encoding. 
+
+A toy dataset is provided under [`data/toy/`](data/toy) which includes `train.txt`, `dev.txt`, and `test.txt`. This example will fit the GPT-2 model on `train.txt`, evaluate perplexity on `dev.txt`, and do continuation generation using `test.txt` as the context.
+
+Run the following cmd to transform the data into [TFRecord](https://www.tensorflow.org/tutorials/load_data/tf_records) format and perform processing such as truncation, BPE encoding, adding special tokens, etc:
+
+```
+    python prepare_data.py --data_dir data/toy 
+    [--max_seq_length=128]
+    [--tfrecord_output_dir=data/toy] 
+    [--pretrained_model_name=117M]
+```
+
+- `data_dir`: The directory of raw data, wherein data files must be named as 'train.txt', 'dev.txt', or 'test.txt'. It is *not* necessary to provide all three files.
+- `max_seq_length`: The maxium length of sequence after BPE encoding. This includes GPT-2 special tokens that will be automatically added. Longer sequence will be trimmed. 
+- `tfrecord_output_dir`: The output path where the resulting TFRecord files will be put in. Be default, it is set to be the same as `data_dir`. 
+- `pretrained_model_name`: The name of a pre-trained model to load selected in the list of: `117M`, `345M`.
+
+The above cmd will output TFRecord files in the specified output directory. E.g., if `train.txt` is provided under `data_dir`, the output file `train.tf_record` will be produced under `tfrecord_output_dir`.
```

Comment:
We no longer use `TFRecord` files. Now outputs are saved as pickled files I think?

Reply:
We might need to modify the `doc` in `RecordData`. The description is outdated. @huzecong